common/presenters/table/object-to-table-head.js

```diff
@@ -1,8 +1,11 @@
 const i18n = require('../../../config/i18n')
 
 function objectToTableHead(schemaEl) {
+  const { head } = schemaEl
+  const prop = head.text ? 'text' : 'html'
```

Comment:
Minor detail but the default should really be `text`. Using the `html` property on GOV.UK Design System components essentially uses the `| safe` filter in nunjucks which will output the value unescaped.

It is safer to default to using `text` with these components and only use `html` when we need to.

Reply:
Actually it's a good point, shall amend.